### PR TITLE
Enable Windows tests

### PR DESCRIPTION
**Pull request by ahartikainen**
Let's see what happens.

### CODE REVIEW

**Comment by riddell-stan** [comment]
Looks like it's failing on the parallel sampling. That's not surprising.

Perhaps we could trim the test to ``- python -m pytest ../tests/test_bernoulli -v`` for now?

In the short-term perhaps we can explore releasing a Windows version of httpstan and pystan-next which somehow prevents people from sampling in parallel. That's certainly better than nothing.

**Comment by ahartikainen** [comment]
Now testing should work. There is some numerical accuracy problems on appveyor. Not sure if that is reproducible or not (test works locally).

https://ci.appveyor.com/project/pystan/httpstan/builds/26529945/job/ew32p0j7m583nge7

```
_________________ test_callbacks_writer_parser_sample_writer __________________
    def test_callbacks_writer_parser_sample_writer():
        """Test that callback writer messages are parsed correctly."""
        messages = [
            """sample_writer:["lp__","accept_stat__","y"]""",  # noqa
            """sample_writer:[-3.16745e-06,0.999965,0.00251692]""",
        ]
        sample_fields = ["lp__", "accept_stat__", "y"]  # noqa
        values = json.loads("""[-3.16745e-06,0.999965,0.00251692]""")
    
        parser = httpstan.callbacks_writer_parser.WriterParser()
        observed = [parser.parse(message) for message in messages]
        message_pb = callbacks_writer_pb2.WriterMessage(topic=TopicEnum.Value("SAMPLE"))
        for key, value in zip(sample_fields, values):
            feature = message_pb.feature.add()
            feature.name = key
            feature.double_list.value.append(value)
        expected = [None, message_pb]
>       assert observed == expected
E       assert [None, topic:...251692\n  }\n}\n] == [None, topic: ...251692\n  }\n}\n]
E         At index 1 diff: topic: SAMPLE\nfeature {\n  name: "lp__"\n  double_list {\n    value: -3.167450000000002e-06\n  }\n}\nfeature {\n  name: "accept_stat__"\n  double_list {\n    value: 0.999965\n  }\n}\nfeature {\n  name: "y"\n  double_list {\n    value: 0.00251692\n  }\n}\n != topic: SAMPLE\nfeature {\n  name: "lp__"\n  double_list {\n    value: -3.16745e-06\n  }\n}\nfeature {\n  name: "accept_stat__"\n  double_list {\n    value: 0.999965\n  }\n}\nfeature {\n  name: "y"\n  double_list {\n    value: 0.00251692\n  }\n}\n
E         Full diff:
E         [None,
E         topic: SAMPLE
E         feature {
E         name: "lp__"
E         double_list {
E         -     value: -3.167450000000002e-06
E         ?                    ----------
E         +     value: -3.16745e-06
E         }
E         }
E         feature {
E         name: "accept_stat__"
E         double_list {
E         value: 0.999965
E         }
E         }
E         feature {
E         name: "y"
E         double_list {
E         value: 0.00251692
E         }
E         }
E         ]
..\tests\test_callbacks_writer_parser.py:54: AssertionError

```

**Comment by riddell-stan** [comment]
We can adjust the test so it uses numpy's almost_equal. Glad to have sampling working on Windows.

**Comment by ahartikainen** [comment]
So we could use `re.sub` and `re.findall` to compare text structure and numbers.

**Comment by riddell-stan** [comment]
Once #195 is merged we can rebase this and run all the tests. Would be wonderful if it worked.

**Comment by riddell-stan** [comment]
Let's rebase this and run all the tests?

File: .appveyor.yml [hunk 1]
```diff
@@ -38,5 +38,5 @@ build: false
 test_script:
   - mkdir empty_folder
   - cd empty_folder
-  # eventually `- python -m pytest ../tests`
   - python -c "import httpstan"
+  - python -m pytest ../tests --deselect tests/test_bernoulli.py::test_bernoulli_parallel -v
```
Comment (riddell-stan) [inline .appveyor.yml]: Let's run all the tests?

Comment (ahartikainen) [reply .appveyor.yml]: Yes, I will change this after the "current test set"  works. So in 30min or so.

**Comment by ahartikainen** [comment]
This is great!

**Pull request merged**